### ISSUE TITLE
Movies & TV shows?

### ISSUE THREAD

**Issue by PGR-14** [CleverRaven/Cataclysm-DDA#74394]
### Is your feature request related to a problem? Please describe.

We've got books, but you know what would go well with them? Movies.

### Solution you would like.

Basically add in stuff like CDs & saved movies on computers, would be faster than reading a book, but would give less knowledge (Most books would go in detail as opposed to TV shows showing the gist of it.), but also much higher morale.

Thought of two ways to express this, either through CDs (W/ Laptops & players) or through regular old USB sticks & memory cards.

**Comment by Mithrite** [comment]
Put one vote down for memory cards as the "publishing medium".  It's basically like a Nintendo Switch game cart, and feels like it fits in more easily than an optical disk.

**Comment by dominic-dimico** [comment]
As a player, I'd love to see a function for entertainment centers.  Even if all they have to do is emit light and music providing a morale boost equivalent to the stereo system, that would be great.  But I also imagine them being a workaround for illiterate characters and book-haters, provided they can bootstrap their electronics proficiency to the point of hooking a TV up.  Cooking and home improvement shows abound, for example.

I also reckon that an entertainment center would make a fantastic trap for intelligent enemies who aren't already familiar with it, such as Mi-Gos.  I imagine they'd be quite entranced by TVs, since their _modus operandi_ is to learn and replicate human speech patterns so they can trick humans.

**Comment by PGR-14** [comment]
I agree, but entertainment centers aren't electrical, they're kind of like cabinets.
